app/Image/MediaFile.php

```diff
@@ -2,12 +2,13 @@
 
 namespace App\Image;
 
-use App\Exceptions\Internal\LycheeLogicException;
 use App\Exceptions\MediaFileOperationException;
+use App\Exceptions\MediaFileUnsupportedException;
+use App\Models\Configs;
 use Illuminate\Http\UploadedFile;
 
 /**
- * Interface MediaFile.
+ * Class MediaFile.
```

Comment:
Yes and no. We need a short first line, because the first line of the PHPdoc comment is used as the title (during auto-completion by the IDE and if we will ever use it, in the generated PHP documentation). The following sentence is rather long:

```php
/**
 * Class MediaFile.
 *
 * This class abstracts from the differences of files which are provided
 * through a Flysystem adapter and files outside Flysystem.
 *
```

You definitely don't want the second sentence as a caption inside a table or similar.

But I agree that the caption should be a little bit more than just the repeated class name. I was simply too lazy to come up with another sentence which is longer than that but shorter than the following.